### PR TITLE
feat: integrate tailwind-config-viewer

### PR DESCRIPTION
**Pull request by rogden**
@Atinux This PR (recreated as a draft PR) integrates https://github.com/rogden/tailwind-config-viewer to replace the current colors UI.

It is served via serverMiddleware so it does conflict with the app CSS. Adding the serverMiddleware only works outside of the build:before hook for some reason. When called within the hook, it results in a 404. Feel free to change as needed and reach out with any questions.

### CODE REVIEW

File: lib/module.js [hunk 1]
```diff
@@ -123,6 +101,18 @@ module.exports = async function (moduleOptions) {
       }
     }
   })
+
+  /*
+   ** Add /_tailwind UI
+   */
+  const tailwindConfigViewer = require('./serverMiddleware/tailwindConfigViewer')
```
Comment (Geminii) [inline lib/module.js:108]: `/_tailwind` doesn't need to be expose only if options.exposeConfig is true ?

Comment (rogden) [reply lib/module.js:108]: _tailwind exposes the UI for viewing the config values. It doesn't rely on the exposed config file, only the tailwindConfig object that is passed into the `tailwindConfigViewer()` invocation. 

It likely needs to be wrapped in `if (nuxt.options.dev)` or a `if (nuxt.options.dev && options.viewer)` though.

Comment (Geminii) [reply lib/module.js:108]: Sorry, it's a misunderstanding on my part about `exposeConfig` parameter. 
Is there an interest in exposing the ui in production mode ? 
Maybe it make sense to purpose this option to disable it if needed 👍

Comment (rogden) [reply lib/module.js:108]: Should we default viewer to true? 

@Atinux I noticed a `ui` option in the [default options object](https://github.com/nuxt-community/tailwindcss-module/blob/master/lib/module.js#L16). Was that mean't to be the flag for whether the Tailwind colors ui should be enabled? If so, we could use it vs using a `viewer` option.

Comment (Atinux) [reply lib/module.js:108]: I think we should rename it to `viewer` (`ui` is not even used at the moment).

**Pull request merged**